src/styles/pages/admin/_content-pages.scss

```diff
@@ -47,4 +49,27 @@
 	> .c-content-page-preview {
 		margin-top: $spacer-lg;
 	}
+
+	.c-user-group-select {
+		.c-checkbox-group {
+			border: 1px solid grey;
+			border-radius: $input-border-radius;
+			background-color: $white;
+			padding: 1px;
+
+			.c-checkbox {
+				display: block;
+				padding: 1.5rem;
```

Comment:
Removing the `1px` padding on the parent and upping the child's padding to `$spacer-sm` should have about the same effect right?

Reply:
I've changed it to a $spacer-sm.
The 1px padding is needed to avoid the hover effect to cut into the parent border. like this:
![image](https://user-images.githubusercontent.com/1710840/191786348-d459cede-a808-4671-92bd-79f870cb564e.png)